test/Market.t.sol

```diff
@@ -1909,7 +1903,6 @@ contract Market_validates_certificate_amount is UpgradeableUSDCMarket {
         numberOfNRTsToPurchase,
         _namedAccounts.supplier
       );
-      vm.stopPrank();
```

Comment:
Are these being removed b/c these `stopPrank()` is being used unnecessarily? 

ex if like 1901|1895 used `vm.startPrank(...)` we would need this line, but since it doesn't, the call to `vm.stopPrank()` is doing nothing?

Reply:
Correct, and in fact it failed the test when in clouded because there was no prank running. Not sure if this is due to a new forge version or something else, but either way it was always doing nothing.